site/src/docs/patterns/navigation/building-navigation.mdx

```diff
@@ -0,0 +1,522 @@
+
+<ul style="margin-bottom:0;">
+  <li>
+    The learn more about the <strong>heading hierarchy</strong>, please refer to
+    <InternalLink href="/foundation/design-tokens/typography#heading-sizes">the Typography page.</InternalLink>
+  </li>
+  <li>
+    The learn more about the <strong>SideNavigation</strong> component, please refer to
+    <InternalLink href="/components/side-navigation">the component page.</InternalLink>
+  </li>
+  <li>
+    The learn more about the <strong>Tabs</strong> component, please refer to
+    <InternalLink href="/components/tabs">the component page.</InternalLink>
+  </li>
+</ul>
+
+### 7. Footer
+
+The footer is placed at the bottom of the site and includes supplementary information and links, such as contact details, legal information, social media icons, and secondary navigation elements. The footer helps users find additional resources and reinforces the overall site structure by providing a consistent and accessible navigation area on every page.
+
+<PlaygroundPreview>
+  <Footer title="Helsinki Design System" footerAriaLabel="HDS Footer">
+    <Footer.Navigation navigationAriaLabel="HDS Footer navigation">
+      <Footer.Item label="Item" />
+      <Footer.Item label="Item" />
+      <Footer.Item label="Item" />
+      <Footer.Item label="Item" />
+      <Footer.Item label="Item" />
+      <Footer.Item label="Item" />
+      <Footer.Item label="Item" />
+      <Footer.Item label="Item" />
+    </Footer.Navigation>
+    <Footer.Utilities backToTopLabel="Back to top">
+      <Footer.SoMe>
+        <Footer.Item icon={<IconFacebook />} />
+        <Footer.Item icon={<IconTwitter />} />
+        <Footer.Item icon={<IconInstagram />} />
+        <Footer.Item icon={<IconYoutube />} />
+        <Footer.Item icon={<IconTiktok />} />
+      </Footer.SoMe>
+      <Footer.Item label="Contact us" />
+      <Footer.Item label="Give feedback" />
+    </Footer.Utilities>
+    <Footer.Base copyrightHolder="Copyright" copyrightText="All rights reserved">
+      <Footer.Item label="Link" />
+      <Footer.Item label="Link" />
+      <Footer.Item label="Link" />
+      <Footer.Item label="Link" />
+      <Footer.Item label="Link" />
+    </Footer.Base>
+  </Footer>
+</PlaygroundPreview>
+
+### Focus order & keyboard navigation
+
+Building a correct focus order is essential to ensure accessibility and provide a good user experience for all users, including those using keyboard navigation or assistive technologies. It's crucial to test your focus order using a keyboard alone to ensure it works as expected.
+
+Focus order, or tab order, refers to the sequence in which interactive elements on a webpage or application receive focus when a user navigates using a keyboard. Here's a general guideline on how to build the focus order for keyboard navigation:
+
```

Comment:
One more thing, should we bring the example with important links here as well, as a text or as a image?
So it would be main navigation, breadcrumbs, sidenavigation, important links, main content, possible right panel and then footer ?